PIL/Image.py

```diff
@@ -2206,6 +2206,8 @@ def fromqpixmap(im):
     # first two members of shape are set to one
     # ((1, 1), "|b1"): ("1", "1"), # broken
     ((1, 1), "|u1"): ("L", "L"),
+    ((1, 1), "<u2"): ("L", "I;16"),
+    ((1, 1), ">u2"): ("L", "I;16B"),
     ((1, 1), "|i1"): ("I", "I;8"),
```

Comment:
Additionally, I believe the "I" means that it's a color image, but in this case it's not. I don't quite understand all of these special strings that Pillow uses, so maybe I'm wrong here. I do believe this code is working as shown by my testcase. You can fully roundtrip a uint16 image with no dataloss.